lib/Rex/CLI.pm

```diff
@@ -30,8 +30,6 @@ use Rex::TaskList;
 use Rex::Logger;
 use YAML;
 
-use Data::Dumper;
-
```

Comment:
Is there any reason why Data::Dumper is also removed in this context about Rex::Template::* deprecation?

Reply:
This module is unused in `CLI.pm`.